es_ES.json

```diff
@@ -1602,6 +1603,7 @@
 	"property.origin.saveButton": "Salvar",
 	"property.origin.server": "Servidores *",
 	"property.origin.server.info": "Please enter at least one valid server. If you do not specify HTTP and HTTPS ports, the respective defaults 80 and 443 will be used.",
```

Comment:
Is this text used? It's odd that some entries in the language file were not translated.

Reply:
Yes...Some texts that were added prior to active internationalization efforts aren't translated. I don't plan to update those texts as part of this task.
Can do that in the future.

I believe we are currently mainly focused on English and Chinese language for accuracy purposes.